### PR TITLE
Added info about global functions

### PR DESCRIPTION
**Pull request by grelltrier**
When generating bindings for their own project, users can have global functions. Added info on how to deal with them.

### CODE REVIEW

File: docs-src/tutorial/gir_tutorial.md [hunk 1]
```diff
@@ -314,6 +314,10 @@ Another `gir` run:
 > gir
 ```
 
+There is also a third option you can encounter called [NOT GENERATED FUNCTION]. If you see this in your own project, it means you have global functions.
+To generate the code for all of them, add "YourGirName.*" to the generate = [ ] array. You also need to re-export them, so add
```
Comment (sdroege) [inline docs-src/tutorial/gir_tutorial.md:318]:
```suggestion
To generate the code for all of them, add "YourGirName.*" to the `generate = [ ]` array. You also need to re-export them, so add 
```

File: docs-src/tutorial/gir_tutorial.md [hunk 2]
```diff
@@ -314,6 +314,10 @@ Another `gir` run:
 > gir
 ```
 
+There is also a third option you can encounter called [NOT GENERATED FUNCTION]. If you see this in your own project, it means you have global functions.
```
Comment (sdroege) [inline docs-src/tutorial/gir_tutorial.md:317]:
```suggestion
There is also a third option you can encounter called `[NOT GENERATED FUNCTION]`. If you see this in your own project, it means you have global functions.
```

**Review by sdroege — APPROVED** [review]
Makes sense to me, apart from the formatting :) Thanks!

**Comment by grelltrier** [comment]
This is also mentioned in the draft to rewrite the gir tutorial

**Pull request closed without merging**